src/app/modules/edit-user-nickname-modal/edit-user-nickname-modal.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, inject, model } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { IonButton, IonIcon, IonInput, IonItem, IonList, ModalController } from '@ionic/angular/standalone';
+import { TranslatePipe } from '@ngx-translate/core';
+import { addIcons } from 'ionicons';
+import { close } from 'ionicons/icons';
+
+@Component({
+  selector: 'app-edit-user-nickname-modal',
+  templateUrl: './edit-user-nickname-modal.component.html',
+  styleUrls: ['./edit-user-nickname-modal.component.scss'],
+  imports: [FormsModule, IonButton, IonIcon, IonInput, IonItem, IonList, TranslatePipe],
+})
+export class EditUserNicknameModalComponent {
+  modalController = inject(ModalController);
+
+  nickName = model<string>();
```

Comment:
Det er ikke så farlig, men i dette tilfellet tror jeg ikke du trenger en model her. Tror du kan klare deg med en helt vanlig string variabel.

Reply:
Ah, ser nå at nickName også er input for modalen. Da er det fint å bruke model her såklart 👍